packages/theming/src/types/index.ts

```diff
@@ -75,14 +75,24 @@ export interface IGardenTheme {
   };
   colors: {
     base: 'light' | 'dark';
-    background: string;
-    foreground: string;
     primaryHue: string;
     dangerHue: string;
     warningHue: string;
     successHue: string;
     neutralHue: string;
     chromeHue: string;
+    variables: {
+      dark: {
+        background: Record<string, string>;
+        border: Record<string, string>;
+        foreground: Record<string, string>;
+      };
+      light: {
+        background: Record<string, string>;
+        border: Record<string, string>;
+        foreground: Record<string, string>;
+      };
+    };
```

Comment:
How are you feeling about the `variables` property name? I was thinking about the [CSS naming](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) the other day (e.g., `scheme` from `prefers-color-scheme`). I don't feel particularly strongly, just curious if that's come up for you at all.

Reply:
Also thinking about how this writes in our components:

- `p.theme.colors.variables.light.foreground`
- `p.theme.colors.scheme.light.foreground`
- etc

😅

Reply:
> How are you feeling about the `variables` property name? I was thinking about [CSS naming](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme) the other day (e.g., `scheme` from `prefers-color-scheme`). I don't feel particularly strongly, just curious if that's come up for you at all.

I don't like `scheme` as this object is not a scheme – it's a collection of variables. We're leaning into the design language here and making room for additional themes such as "lightHighContrast", "darkDimmed", etc.

Reply:
> Also thinking about how this writes in our components:
> 
> * `p.theme.colors.variables.light.foreground`
> * `p.theme.colors.scheme.light.foreground`
> * etc
> 
> 😅

We will never reference values in this manner. It should make more sense when the `getColor` PR lands, but the function call will be something like `getColor({ theme, variable: 'foreground.default' });`

Reply:
> I don't like scheme as this object is not a scheme

Makes sense, especially with high contrast and other themes in the variable list. 👌🏻